examples/image_classification-tf.ipynb

```diff
@@ -0,0 +1,7883 @@
+{
```

Comment:
<p>Why there's no <code>val_accuracy</code>?  <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/notebooks/pull/202/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='60'/>

Reply:
<p>I've edited the metric callback to print out <code>val_accuracy</code>  at the end of epoch. This won't show train accuracy however. I believe this would require another callback to be added for computing the metric on train (is this right @Rocketknight1 ?) </p>

Reply:
Thanks!